content/reference/sql/sql-support/miscellaneous.md

```diff
@@ -89,8 +88,7 @@ SELECT * from information_schema.tables;
 +-------------+------------+-------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 ```
 
-Alternatively, the `dolt_statistics` table exposes histogram
-information:
+Statistics are persisted in database's chunk store in a `refs/stats` ref stored separately from the commit graph. Each database has its own statistics store. The contents of the `refs/stats` reflect a single point-in-time, and are un-versioned. The contents of this ref in the current database can be inspected with the `dolt_statistics` system table. 
```

Comment:
Does this mean that you can't track stats separately for multiple branches? You should explicitly mention whether this is the case or not.

Reply:
yeah, there's probably a way to do it with `call dolt_stats_restart() as of 'branch'`, and then choose the ref based on the specific branch name before falling back to the default. But probably not until someone asks for it.